norma/views.py

```diff
@@ -69,16 +69,16 @@ def get_queryset(self):
             periodo_final = datetime.strptime(
                 kwargs['periodo_final'],
                 '%d/%m/%Y').strftime('%Y-%m-%d')
```

Comment:
Trechos como esse assumem que as strings estão sempre codificadas como d/m/Y, mas e se o locale for m/d/Y?